spec/coffee/directives/api/utils/_async.spec.coffee

```diff
@@ -2,21 +2,23 @@ describe "_async", ->
   rootScope = null
   timeout = null
   q = null
+  browser = null
 
   digest = (fn, times = 1) =>
     fn()
     if times
       _.range(times).forEach -> # i would like to say that it sucks that I have to do this.. (angular)
-        timeout?.flush()
+        timeout?.flush() if browser.deferredFns?.length
```

Comment:
So `verifyNoPendingTasks()` in specs throughs errors.
```
Error: Deferred tasks to flush (1): {id: 0, time: 1}
```

So this appears to be a no go.